### DISCUSSION TITLE
Deprecate translations?

### DISCUSSION THREAD

**Discussion by sandstrom** [libvips/libvips#2079] [Ideas]
Just a thought, but are translations really needed in libvips? Most libraries would not be concerned with translations and leave that to the consumer.

Maybe libvips could shed some weight by dropping them and going english-only.

https://github.com/libvips/libvips/tree/master/po

*(I'm saying this as someone who's native language isn't english)*

**Comment by jcupitt** [comment]
They are used at the CLI, eg.:

```
john@yingna ~ $ export LC_ALL="de_DE.UTF-8"
john@yingna ~ $ export LANG="de_DE.UTF-8"
john@yingna ~ $ export LANGUAGE="de_DE:de:en_US:en"
john@yingna ~ $ vips
Aufruf:
  vips [OPTION …] [AKTION] [OPTIONEN] [PARAMETER] - VIPS-Treiberprogramm

Hilfeoptionen:
  -h, --help                    Hilfeoptionen anzeigen

Anwendungsoptionen:
...
```

So they can be handy. Some brave person does need to write the translation file, of course :(

**Comment by sandstrom** [comment]
I would guess that ~90% of CLI tools aren't translated, since English is the de-facto language for all computer systems.

I'd vote for deprecating them! But it's just an idea, you should only do that if you think it'll make the project easier to maintain.

Either way, libvips is awesome! 🏅

**Comment by jcupitt** [comment]
One other factor is that these messages appear in the various libvips GUIs as well, and many people do expect these to be internationalised. So removing libvips i18n would also remove a lot of GUI i18n.

**Reply by sandstrom** [reply]
Maybe separating the two would make sense?

I don't know the scope of the GUI (I've never used it), but it would seem like a good separation of concern to break out any GUI from libvips, since my understanding is that libvips is mostly used non-interactively.

Such a broken-out GUI project (which would obviously depend on libvips) would then host its own translation files.

**Reply by sandstrom** [reply]
Just a friendly ping 🎄 

I still think you should consider dropping translations, at least for the CLI. To reduce maintenance burden.
